ui/appui-react/src/appui-react/UiFramework.ts

```diff
@@ -160,6 +162,9 @@ export class UiFramework {
     if (frameworkStateKey && store)
       UiFramework._frameworkStateKeyInStore = frameworkStateKey;
 
+    if (startInUi1Mode)
+      UiFramework.store.dispatch({ type: ConfigurableUiActionId.SetFrameworkVersion, payload: "1" });
```

Comment:
Why was this added to core and not something DR could call after initialing UiFramework themselves?

Reply:
@aruniverse - We saw strange crashes in UI components if this was done after the initalize promise resolved.  I did not see any errors if the update was done during initialize processing. You can make the change once the entire frontstage is rendered without issue.